numerics.rkt

```diff
@@ -1,12 +1,26 @@
 #lang racket/base
 
-;; TODO import the rest of numerics/functions
-;;      rename conflicting identifiers (between elliptic & elliptic-flo, etc)
+(require
+ mechanics/private/numerics/functions/bessel
+ mechanics/private/numerics/functions/bessjs
+ mechanics/private/numerics/functions/elliptic
+ (prefix-in flo: mechanics/private/numerics/functions/elliptic-flo))
 
 (provide
-  (all-from-out mechanics/private/numerics/functions/bessel)
-)
-
-(require
-  mechanics/private/numerics/functions/bessel
-)
+ (all-from-out mechanics/private/numerics/functions/bessel)
+ (all-from-out mechanics/private/numerics/functions/bessjs)
+ 
+ (rename-out
+  [bessj₀ bessj0]
+  [bessj₁ bessj1]
+  [bessy₀ bessy0]
+  [bessy₁ bessy1]
+  [bessh₀ bessh0]
+  [bessh₁ bessh1])
+ 
+ (rename-out
+  [Rf Carlson-elliptic₁]
+  [Rf Carlson-elliptic-1]
```

Comment:
Can we make all the ascii versions be `fn_N`?

Mostly, I want to be consistent. But also I think underscores are better than hyphens or nothing at all.